frontend/src/routes/ApplicationForm/FormStructure.js

```diff
@@ -42,7 +42,7 @@ const FormStructure = ({
           {myApplication && (
             <StyledSpan bold>
               <Moment format="dddd Do MMMM , \k\l. HH:mm:ss">
-                {myApplication.time_sent}
+                {myApplication.created_at}
```

Comment:
Shouldn't this be `updated_at`? As it's the last time the application was registered

Reply:
Yup, fixed